### ISSUE TITLE
DOC: fix SA01 errors in docstrings 

### ISSUE THREAD

**Issue by jordan-d-murphy** [pandas-dev/pandas#57417]
Pandas has a script for validating docstrings:

https://github.com/pandas-dev/pandas/blob/44c50b20e8d08613144b3353d9cd0844a53bd077/ci/code_checks.sh#L416-L1426

Currently, some methods fail the SA01 check.

The task here is:

- take 2-4 methods
- run: scripts/validate_docstrings.py --format=actions --errors=SA01 `method-name`
- check if validation docstrings passes for those methods, and if it’s necessary fix the docstrings according to whatever error is reported
- remove those methods from code_checks.sh
- commit, push, open pull request

Please don't comment take as multiple people can work on this issue. You also don't need to ask for permission to work on this, just comment on which methods are you going to work.

If you're new contributor, please check the [contributing guide](https://pandas.pydata.org/docs/dev/development/contributing.html)

**Comment by jordan-d-murphy** [comment]
Addresses https://github.com/pandas-dev/pandas/pull/57354

**Comment by jordan-d-murphy** [comment]
opened up a fix for `pandas.ArrowDtype`

**Comment by jordan-d-murphy** [comment]
opened a fix for 
```
pandas.BooleanDtype
pandas.StringDtype
```

**Comment by jordan-d-murphy** [comment]
opened a fix for the following: 
```
pandas.Categorical.as_ordered
pandas.Categorical.as_unordered
pandas.CategoricalIndex.as_ordered
pandas.CategoricalIndex.as_unordered
pandas.Series.cat.as_ordered
pandas.Series.cat.as_unordered
```

**Comment by jordan-d-murphy** [comment]
Hi all, if [CI: speedup docstring check consecutive runs #57826](https://github.com/pandas-dev/pandas/pull/57826) gets merged in, I might be reworking our approach here; this would look like closing the following issues: 


> [DOC: fix GL08 errors in docstrings](https://github.com/pandas-dev/pandas/issues/57443)
> [DOC: fix PR01 errors in docstrings](https://github.com/pandas-dev/pandas/issues/57438)
> [DOC: fix PR07 errors in docstrings](https://github.com/pandas-dev/pandas/issues/57420)
> [DOC: fix SA01 errors in docstrings](https://github.com/pandas-dev/pandas/issues/57417)
> [DOC: fix RT03 errors in docstrings](https://github.com/pandas-dev/pandas/issues/57416)
> [DOC: fix PR02 errors in docstrings](https://github.com/pandas-dev/pandas/issues/57111)

And opening a new issue to address these based on the new approach. 

tl;dr 

the work can still be done, but probably under a new ticket once  #57826 is merged in

**Comment by jordan-d-murphy** [comment]
Closed by:
[CI: speedup docstring check consecutive runs #57826](https://github.com/pandas-dev/pandas/pull/57826)
[CI: Better error control in the validation of docstrings #57879](https://github.com/pandas-dev/pandas/pull/57879) 

Opening a new issue to fix docstring errors.

**Comment by jordan-d-murphy** [comment]
Opened [DOC: Enforce Numpy Docstring Validation (Parent Issue) #58063](https://github.com/pandas-dev/pandas/issues/58063) as a parent issue for fixing docstrings based on the refactoring in code_checks.sh 

Feel free to swing by and help out! 🙂
